wikimedia/wikipedia-ios
# Add Tachelhit Tifinagh script variant, update variants to use BCP47 formatted codes

**Phabricator:** https://phabricator.wikimedia.org/T333596 & https://phabricator.wikimedia.org/T338079

### Notes
* This PR adds the Tachelhit Tifinagh script variant to the app and updates our language variant codes to the BCP47 format as the mobileapps service 

### Test Steps
- To test the new Tachelhit script
1.  Run the app on main, and add the Tachelhit language to your languages
2. Change to this branch. You should see this panel when first opening the app.
3. Add the new variant to your languages and confirm the explore feed correctly displays the Tifinagh script

- To test the new variant codes
1. On settings > My languages, add the languages with variants (Chinese, Tachelhit, Serbian, Crimean Tatar, Gan, Inuktitut, Kazakh, Kurdish, Tajik, Uzbek)
4. Confirm that all variants are correctly displayed on the explore feed
PS: I had some persistence issues on the explore feed when removing/adding languages. Deleting the app and then installing it again resolved it, I'm not sure what was causing it

### Screenshots/Videos

<img src="https://github.com/wikimedia/wikipedia-ios/assets/31116510/bc634bc6-3ac7-4fb7-a40f-42754483da93" width="295" height="639">
